src/modules/associated-test-modal/associated-test-modal.tsx

```diff
@@ -56,13 +56,15 @@ export const AssociatedTestModal = () => {
             skeleton={isSkeleton}
             title={packageName}
             data-test="associated-tests:package-name"
+            tw="lowercase"
           >
             {isSkeleton ? "" : packageName || "-"}
           </MethodInfoValue>
           <MethodInfoValue
             skeleton={isSkeleton}
             title={testClassName}
             data-test="associated-tests-modal:test-class-name"
+            tw="first-letter:uppercase"
```

Comment:
Are you sure that here should be uppercase for first letter? As i know All class names in Java should be in capitalized by default.  AAnd it meens that we don`t need to use this style